site/ko/blog/new-in-devtools-94/index.md

```diff
@@ -0,0 +1,180 @@
+---
+layout: "layouts/blog-post.njk"
+title: "DevTools 의 새로운 기능 (Chrome 94)"
+authors:
+  - jecelynyeen
+date: 2021-08-24
+updated: 2021-08-24
+description:
+  "DevTools 를 원하는 언어로 사용할 수 있습니다. 그 외 새 Nest Hub 기기들, 새 CSS 컨테이너 쿼리 배지 등 많은 새 기능들을 확인해보세요."
+hero: 'image/dPDCek3EhZgLQPGtEG3y0fTn4v82/js1uKEulNBGj0PoJkqZq.jpg'
+alt: ''
+tags:
+  - new-in-devtools
+  - devtools
+  - chrome-94
+---
+
+*이 게시글의 번역에는 [최원영](https://www.linkedin.com/in/toruchoi)님이 참여하였으며, [이지웅](https://bit.ly/JiwoongLeePortfolio)님과 [조은](https://developers.google.com/community/experts/directory/profile/profile-eun-cho)님이 리뷰를 맡아 주셨습니다.*
+
+
+{% include 'partials/devtools/kr/banner.md' %}
+
+## DevTools 를 이제 여러분들이 선호하는 언어로 사용하세요 {: #localized }
+
+Chrome DevTools는 80개 이상 언어를 제공하여, 여러분들의 기호에 맞는 언어를 골라서 사용할 수 있습니다.
+
+[Settings](/docs/devtools/customize/#settings) 메뉴를 열고, **Preferences** > **Language** 드롭다운 메뉴에서 여러분들이 원하는 언어를 선택한 다음  DevTools 를 새로고침해 줍니다.
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/eozpCcjmnn7zwya9zXu6.png", alt="Settings > Preferences 에서 언어를 선택하세요", width="800", height="494" %}
+
+{# https://chromium.googlesource.com/chromium/src/+/58abfbcdddae27fb43c17f43dbcc197f2570b5a5 #}
+
+Chromium issue: [1163928](https://crbug.com/1163928)
+
+## 기기 목록에 Nest Hub 가 새로 추가되었습니다 {: #nest-hub }
+
+이제  [기기 모드](/docs/devtools/device-mode/) 에서 Nest Hub 및 Nest Hub Max Dimension 으로 시뮬레이션할 수 있습니다.
+
+[Toggle Device Toolbar](/docs/devtools/device-mode/#viewport) 를 클릭한 다음,&nbsp; {% Img src="image/admin/9FiBHFCzfPgP8sy6LMx7.png", alt="토글 디바이스 툴바", width="20", height="22" %}&nbsp;,  기기 목록에서  Nest Hub 혹은 Nest Hub Max 를 선택하세요.
+
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/KytKWMiC4cbFfVUOBzlm.png", alt="기기 모드의 Nest Hub device", width="800", height="549" %}
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/d13f911f7d98751cce659898936511b5ccda96cd #}
+
+Chromium issue: [1223525](https://crbug.com/1223525)
+
+
+## 프레임 상세 화면에서 Origin trials 가 제공됩니다 {: #origin-trials }
+
+이제 웹사이트의 [origin trials](/blog/origin-trials/)  에 대한 정보를 애플리케이션 패널 하단의 프레임 상세 화면에서 볼 수 있습니다.
+
+사용자들은 [Origin trials](/blog/origin-trials/) 를 통해 새로운 기능 혹은 실험중인 기능에 액세스할 수 있으며, 이는 사용자들로 하여금 해당 기능이 안정화되기 전에 제한적인 시간 동안 시도해 볼 수 있도록 하기 위함입니다.
```

Comment:
In this context, 'you' is more of the reader of this article which is developers than users, so I'd use '여러분' than '사용자들'. And also I'd like to translate 'access' to Korean word because it's not a unique noun in this context.

suggestion:
여러분은 [Origin trials](/blog/origin-trials/) 를 통해 새로운 기능 혹은 실험중인 기능을 적용해볼 수 있으며, 이는 사용자들로 하여금 해당 기능이 안정화되기 전에 제한적인 시간 동안 시도해 볼 수 있도록 하기 위함입니다.

Reply:
여러분들은 Origin trials 를 통해 새로운 기능 혹은 실험적인 기능을 이들이 안정화되기 전에 제한적인 시간 동안 시도 및 적용해 볼 수 있습니다. 
^ How about this?  I will apply this. Please feel free to suggest :)